app/renderer/components/navbar.js

```diff
@@ -66,6 +66,7 @@ module.exports = {
         },
         onTabSelected(index) {
             AppStatus.activeSuite = index;
+            AppStatus.totalTasks = this.suites[index].length;
```

Comment:
With this, I understand you only count the tasks of the current suite. Should total tasks count every task in gaucho?

(this can easily be changed here with a [reduce](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) function

Reply:
And the value wouldn't change on each tab. We should discuss the exact use case of the variable `totalTasks` on an issue

Reply:
discuss: https://github.com/angrykoala/gaucho/issues/132